### ISSUE TITLE
CREMAD实验

### ISSUE THREAD

**Issue by JeromeQAQ1** [GeWu-Lab/BML_TPAMI2024#3]
使用了OGM-GE的数据处理方法，在MMPareto，InfoReg等方法上使用默认参数结果与论文结果差距在5~8%，本文实验结果为65.73%，使用了别的issue里面提到的换lr，换sgd和batch_size，仅提升不到2%，与论文结果差距明显

**Comment by echo0409** [comment]
你好！感谢关注。期刊版本的实验采用了lr_scheduler，adamw等调整，精度比较高。同时数据预处理方式和期刊版本也略有差别。其他会议版本一般使用的默认配置是sgd，无scheduler。但是每次论文的实验都是重新跑的，细节不完全一致，不同论文间整体精度幅度有差异请见谅。

但是在同一版本的论文中，我们在进行对比时是保证表格中所有对比方法采用的这些超参数是完全一致的，保证了对比公平。

期刊版本以下参数供参考：

batch_size": 32,
    "epochs": 100,
    "optimizer": "Adamw",
    "learning_rate": 0.001,
    "lr_decay_ratio": 0.1,
    "lr_decay_step": 20,
    "use_modulation": true,
    "modulation": "OGM_GE",
    "modulation_starts": 0,
    "modulation_ends": 80,
    "alpha": 2.6,
    "fusion_method": "concat",

optimizer=torch.optim.AdamW(model.parameters(),lr=cfg.learning_rate,weight_decay=0.01) scheduler=torch.optim.lr_scheduler.StepLR(optimizer,step_size=cfg.lr_decay_step,gamma=cfg.lr_decay_ratio)
